src/main/java/com/epam/esm/hateoas/model/UserModel.java

```diff
@@ -0,0 +1,20 @@
+package com.epam.esm.hateoas.model;
+
+import com.epam.esm.domain.Order;
+import com.epam.esm.domain.Role;
+import com.epam.esm.domain.User;
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import jakarta.persistence.*;
```

Comment:
Don't forget about this *

Reply:
I've just figured out that my IDE does it implicitly and I can't fix it. I'll try to google it